tests/webidl/post.js

```diff
@@ -76,11 +76,11 @@ try {
 } catch(e) {}
 console.log(succeeded);
 
-TheModule.Child2.prototype.printStatic(); // static calls go through the prototype
+TheModule.Child2.printStatic(); // static calls do not go through the prototype
```

Comment:
would this still work with `.prototype` as well? if yes, then we should test both. if not, then it's a breaking change, and maybe not worth it.

Reply:
The old approach should work with or without `.prototype`, I think, so if we decide to not change this, maybe we can at least add testing and docs for not writing `.prototype`

Reply:
When I try and call something like `MyModule.CConfig.SetStaticOption(...)` I get (in Chrome) `Uncaught TypeError: `MyModule.CConfig.SetStaticOption is not a function.

I would have thought for this to work I would have needed to instantiate a `CConfig` option. Right?

The proposed change still works like `MyModule.CConfig.prototype.SetStaticOption(...)`, but can now also work like `MyModule.CConfig.SetStaticOption(...)`.

Reply:
I see, thanks, sounds good.

Reply:
I think that leaves testing - as mentioned earlier, please both both notations that work after this change.

Reply:
I updated the tests to test both access methods, but the below line (`TheModule.Child2.prototype.printStatic();`) seems to fail on node in `test-uvwxyz`. It looks like it must be throwing? I don't have any node experience, any ideas what could be going wrong?

Also this appears to have increased the number of exports by one in the `test-upstream-other` tests. But I'm not really sure what the significance of that is, or why my change affected that?

Any pointers?